scipy/stats/_stats_py.py

```diff
@@ -274,11 +274,8 @@ def gmean(a, axis=0, dtype=None, weights=None):
         Axis along which the geometric mean is computed. Default is 0.
         If None, compute over the whole array `a`.
     dtype : dtype, optional
-        Type of the returned array and of the accumulator in which the
-        elements are summed. If dtype is not specified, it defaults to the
-        dtype of a, unless a has an integer dtype with a precision less than
-        that of the default platform integer. In that case, the default
-        platform integer is used.
```

Comment:
> Type of the returned array

Not true. If an integer `dtype` is specified, this returns a float.

> unless a has an integer dtype with a precision less than that of the default platform integer. In that case, the default platform integer is used.

Even before we started to make changes, I saw no evidence of this.